CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   during reconnection.
 - Update `startVideoPreviewForVideoInput` to support filters in the preview window.
 - Update browser demo to showcase preview filter capability.
+- Migrated SauceLabs mobile tests to new api. 
```

Comment:
Actually we use simple present in the ` CHANGELOG`, please use `Migrate`

Reply:
Changed!